docker-compose.yml

```
@@ -37,15 +37,13 @@ services:
       - php
 
   db:
-    image: postgres:10-alpine
+    image: mysql:5.6
     volumes:
-      - db-data:/var/lib/postgresql/data:rw
+      - db-data:/var/lib/mysql:rw
     networks:
       - grapes-network
     environment:
-      - POSTGRES_DB=grapes
-      - POSTGRES_PASSWORD=password
-      - POSTGRES_USER=grapes
+      - MYSQL_ROOT_PASSWORD=password
```

```suggestion
      - MYSQL_DATABASE=grapes
      - MYSQL_ROOT_PASSWORD=password
```
il manque le **MYSQL_DATABASE**